WrightTools/artists.py

```diff
@@ -103,9 +106,20 @@ def _apply_labels(self, autolabel='none', xlabel=None, ylabel=None, data=None, c
             self.set_ylabel(ylabel, fontsize=18)
 
     def _parse_limits(self, zi=None, data=None, channel_index=None, dynamic_range=False, **kwargs):
+        null = kwargs.get('null', 0)
         if zi is not None:
-            vmin = np.nanmin(zi)
-            vmax = np.nanmax(zi)
+            if 'levels' in kwargs.keys():
```

Comment:
should this only be respected if `zi` is not None?

Reply:
Probably?  Can you offer your more informed opinion @untzag @ksunden about how `levels` should be treated when `data is not None`?